pixleesdk/src/main/java/com/pixlee/pixleesdk/PXLClient.java

```diff
@@ -268,7 +247,7 @@ public Map getHeaders() throws AuthFailureError {
         sr.setShouldCache(false);
         this.addToRequestQueue(sr);
         return true;
-    }
+    }*/
```

Comment:
why was this all commented out?  If it's not needed anymore, we should just remove the code instead of commenting it out.